webapp/src/components/tutorial/TutorialContainer.tsx

```diff
@@ -23,6 +23,7 @@ interface TutorialContainerProps {
     hasTemplate?: boolean;
     preferredEditor?: string;
     hasBeenResized?: boolean;
+    hideDone?: boolean;
```

Comment:
I'm a bit torn about this. I see that we created shortcuts for other `tutorialOptions.metadata` fields before, but I'm not sure of the benefit here. Since we're adding `hideDone` to the metadata, could we just grab it from the tutorial options? The `hideDone` flag definitely belongs in the metadata, but it feels a bit different to me than the other metadata fields that we decided to pass directly into the `TutorialContainer`. I don't feel strongly about this, especially for this case, but I wanted to bring it up to have a conversation of if there are things that we want to customize about tutorials in the future, should we add an optional prop to the `TutorialContainer`, or should we leave it in the `tutorialOptions`.

Reply:
I'm okay to just read it from metadata (which is already passed in), so I'll make that change.

I could see a theoretical argument towards making TutorialContainer useable without all our tutorial & tutorial metadata classes, but I don't see that being useful in practicality and it's a moot point anyway since we *also* already pass in the metadata itself.